### PR TITLE
Add missing Subprocess.Checked_Spawn_And_Capture

### PR DESCRIPTION
**Pull request by mosteo**
To complete #252

### CODE REVIEW

File: src/alire/alire-os_lib-subprocess.ads [hunk 1]
```diff
@@ -2,17 +2,25 @@ with Alire.Utils;
 
 package Alire.OS_Lib.Subprocess is
 
-   --  Separate from Alire.OS_Lib because not preelaborable
+   --  Separate from Alire.OS_Lib because not preelaborable.
```
Comment (Fabien-Chouteau) [inline src/alire/alire-os_lib-subprocess.ads]: We usually don't add dots (`.`) for single line comments.

Comment (mosteo) [reply src/alire/alire-os_lib-subprocess.ads]: Understood

**Comment by mosteo** [comment]
> So you removed the progress version.

But it wasn't used anywhere? I thought in fact that you had removed it in your rework :confused:

**Comment by mosteo** [comment]
I think I know what's happened, so indeed I may have removed it involuntarily. Let me do some checks...

**Comment by Fabien-Chouteau** [comment]
When you do `alr build` I think the progress animation is used.

Note that I am personally not a huge fan of this kind of animation, I prefer to see the output.

**Comment by mosteo** [comment]
OK, my preliminary inspection shows that the progress version is no longer used (at least in `alr build`) after the Windows PR. I presume as part of your rework to the Spawn versions (that was my original impression).

:later:

I bisected it to commit 49e217298 in the Windows PR.

That said, the output to console is something that for some time I've thought we might want to rework, so I'm ok with losing this now. The only thing is that right now, for long compilations, alr is eerily quiet... (I was trying with the GNATCOLL crate).

**Comment by Fabien-Chouteau** [comment]
> I bisected it to commit [49e2172](https://github.com/alire-project/alire/commit/49e217298a481850c853fcd70525b49fb76d082f) in the Windows PR.

It was not my intention but OK.

> That said, the output to console is something that for some time I've thought we might want to rework, so I'm ok with losing this now. The only thing is that right now, for long compilations, alr is eerily quiet... (I was trying with the GNATCOLL crate).

Probably for build and actions command we should let the output pass through.

**Pull request merged**